kolibri/plugins/learn/assets/src/views/explore-page/index.vue

```diff
@@ -42,23 +49,66 @@
     $trs: {
       explore: 'Topics',
       navigate: 'Navigate content using headings',
+      all: 'All content',
+      topics: 'Topics',
+      exercises: 'Exercises',
+      videos: 'Videos',
+      audio: 'Audio',
+      documents: 'Documents',
+      html5: 'HTML5 Apps',
+      display: 'Display',
     },
     components: {
       'page-header': require('../page-header'),
       'topic-list-item': require('../topic-list-item'),
       'content-grid-item': require('../content-grid-item'),
       'card-grid': require('../card-grid'),
       'card-list': require('../card-list'),
+      'ui-select': require('keen-ui/src/UiSelect'),
     },
+    data: () => ({
+      selectedFilter: '',
+    }),
     computed: {
+      filterOptions() {
+        const options = [{ label: this.$tr('all'), value: 'all' }];
```

Comment:
To DRY this up a little, I would turn the kinds/labels mapping into data and loop over it. e.g.

```javascript
const kindsLabelMap = {
  [ContentNodeKinds.TOPIC]: 'topics',
...
}

for (const kind of Object.keys(kindsLabelMap)) {
  // if (this.contentsContain(kind)) ...
}
```